cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/heartbeat/StreamWriterHeartbeat.java

```diff
@@ -26,30 +30,28 @@
 public class StreamWriterHeartbeat {
 
   /**
-   * Type of heartbeat, describing at what moment it is sent by a Stream writer.
+   * Type of information that describes one {@link StreamSize} object.
    */
-  public enum Type {
+  public enum StreamSizeType {
     /**
-     * Heartbeat sent during Stream writer initialization.
+     * Size sent during Stream writer initialization.
```

Comment:
This is to help the leader of the stream to keep track of what data is "new" and what data is just coming from the initialization of a new stream service, and hence should not count in the aggregated size that the leader keeps track of.